README.md

```diff
@@ -1,35 +1,42 @@
-# OpenPower Firmware Build Environment
+# OpenPOWER Firmware Build Environment
```

Comment:
Annoyingly "OpenPower" was actually the name of a line of P5 machines... https://en.wikipedia.org/wiki/IBM_OpenPower